spec/features/web_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'date'
```

Comment:
Moved to app.rb since the program itself require it to work and it was implicitly required by some gem